examples/next-app/middleware.ts

```diff
@@ -1,7 +1,8 @@
 import { createI18nMiddleware } from 'next-international/middleware';
 import { NextRequest } from 'next/server';
+import { defaultLocale, locales } from './locales/resources';
 
-const I18nMiddleware = createI18nMiddleware(['en', 'fr'] as const, 'fr');
```

Comment:
Could you revert the changes here and in `resources.ts`? Even if it avoids duplicating, we lose quite a bit of context.

Reply:
Yes I will polish this and remove the example entirely. We can talk about a proper and nicer styled example (so it can be hosted too). I also noticed another bug in the middleware where the search params (?x=y) are not forwarded. I will make a seperate PR for that.